From 79e51857d4ce98dc13b723c874b6793cb68a4254 Mon Sep 17 00:00:00 2001
From: "Artyom V. Poptsov" <poptsov.artyom@gmail.com>
Date: Sat, 22 May 2021 07:26:35 +0300
Subject: [PATCH] tests/sssh-ssshd: Use an ECDSA key for SSSH

* tests/sssh-ssshd.scm: Use an ECDSA key for SSSH as using RSA is prohibited
  by the cryptographic policies in Fedora GNU/Linux.
---
 tests/sssh-ssshd.scm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/sssh-ssshd.scm b/tests/sssh-ssshd.scm
index edb53fc..1dec4ef 100644
--- a/tests/sssh-ssshd.scm
+++ b/tests/sssh-ssshd.scm
@@ -48,7 +48,7 @@
 (define *sssh-cmd*
   (string-append
    %topbuilddir "/examples/sssh.scm"
-   " --identity-file=" %rsakey
+   " --identity-file=" %ecdsakey
    " --port=" (number->string *srv-port*)
    " --known-hosts-file=''"
    " " (inet-ntop AF_INET *srv-address*)
